validator/validate.py

```diff
@@ -31,7 +31,9 @@ def validate(path, format="json",
         The format to return the results in. Defaults to "json". Currently, any
         other format will simply return the error bundle.
     `approved_applications`:
-        Path to the list of approved application versions
+        Path to the list of approved application versions.
```

Comment:
It's still used for now, when the validator is run standalone. Ideally, I'd like to be able to fetch it on demand from AMO, in the future, but at the moment we still need it.